src/rpc/quorums.cpp

```diff
@@ -175,7 +175,8 @@ static RPCHelpMan quorum_list_extended()
     };
 }
 
-static UniValue BuildQuorumInfo(const llmq::CQuorumBlockProcessor& quorum_block_processor, const llmq::CQuorumCPtr& quorum, bool includeMembers, bool includeSkShare)
+static UniValue BuildQuorumInfo(const llmq::CQuorumBlockProcessor& quorum_block_processor, const llmq::CQuorumCPtr& quorum,
+                                bool includeMembers, bool includeSkShare)
```

Comment:
clang format suggestion:
```suggestion
static UniValue BuildQuorumInfo(const llmq::CQuorumBlockProcessor& quorum_block_processor,
                                const llmq::CQuorumCPtr& quorum, bool includeMembers, bool includeSkShare)
```